library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -1220,6 +1224,13 @@ bool libspdm_check_context (void *spdm_context)
         }
     }
 
+    if (((context->local_context.capability.flags &
```

Comment:
@xiaoyuruan has clarified (via https://github.com/DMTF/SPDM-WG/issues/3502#issuecomment-2168793109) that this combination is legal so you can remove the check.

Reply:
Thanks. I have deleted the check.